ttnn/cpp/ttnn/operations/reduction/prod/prod.cpp

```diff
@@ -0,0 +1,138 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+
+#include "prod.hpp"
+#include "prod_nc_op.hpp"
+#include "prod_op_all.hpp"
+#include "ttnn/deprecated/tt_dnn/op_library/auto_format.hpp"
+#include "ttnn/cpp/ttnn/operations/creation.hpp"
+#include "ttnn/operations/data_movement/slice/slice.hpp"
+#include "ttnn/operations/data_movement/permute/permute.hpp"
+#include "tt_numpy/functions.hpp"
+
+
+namespace ttnn {
+
+namespace operations {
+
+namespace reduction {
+
+inline Tensor zeros(
```

Comment:
I think you can reuse this method instead
https://github.com/tenstorrent/tt-metal/blob/main/ttnn/cpp/ttnn/operations/creation.hpp#L74